tests/test_purgatory.py

```diff
@@ -81,7 +81,7 @@ def test_purgatory_with_assets(client, base_ddo_url, events_object, monkeypatch)
     assert published_ddo["purgatory"]["state"] is True
 
     # simulate the passage of time (1 hour until next purgatory update)
-    in_one_hour = datetime.now() + timedelta(hours=1)
```

Comment:
nope, here you don't need it. This is not a nonce-related thing, it's a tool to set the time for the tests. Keep "now" here.